engine/common/host.c

```diff
@@ -107,7 +107,7 @@ Host_NewGame
 qboolean Host_NewGame( const char *mapName, qboolean loadGame )
 {
 	qboolean	iRet;
-
+printf("Host_NewGame(\"%s\", %s)\n", mapName, (loadGame)?"TRUE":"FALSE");

```

Comment:
Better use MsgDev if you want console output. It automatically writes to game console, file and stdout.